backend/src/main.py

```diff
@@ -52,32 +48,70 @@ def post_user(user: UserCreate, db: Session = Depends(get_db)):
     return create_user(db, user)
 
 
-@app.get("/users/", response_model=list[User])
+@app.get("/users/", response_model=list[UserRead])
```

Comment:
Let's remove trailing /. FastAPI handles redirections better that way, in my experience.